### ISSUE TITLE
No DataVector::resize in pysgpp

### ISSUE THREAD

**Issue by valentjn** [SGpp/SGpp#55]
In GitLab by @valentjn on Jan 16, 2018, 08:02

When working in Python, calling `resize` on a `pysgpp.DataVector` throws an error:
```
AttributeError: 'DataVector' object has no attribute 'resize'
```

I think 2d6ee96 by @rappjl broke it... This commit deleted the `resize` method from the `DataVector.i` file because `resize` is now implemented by `std::vector` and not `DataVector`, but this method won't show up in pysgpp then.

Current workaround is to use `DataVector::resizeZero` (slightly slower due to zero initialization).

Can someone confirm this? Would there be any harm in just re-adding `resize` to `DataVector.i`?

**Comment by valentjn** [comment]
In GitLab by @valentjn on Jan 18, 2018, 09:56

Re-adding `resize` to `DataVector.i` seems to work for me.

**Comment by valentjn** [comment]
In GitLab by @valentjn on Jan 19, 2018, 15:37

mentioned in commit dddc5bc2711b746953affb972865721144056ba4

**Comment by valentjn** [comment]
In GitLab by @valentjn on Jan 26, 2018, 16:57

mentioned in merge request !35

**Comment by valentjn** [comment]
In GitLab by @valentjn on Jan 26, 2018, 17:42

mentioned in commit e475f612a64026863826aa80eedfb7513cc97953

**Comment by valentjn** [comment]
In GitLab by @valentjn on Jan 26, 2018, 18:40

closed via commit e475f612a64026863826aa80eedfb7513cc97953

**Comment by valentjn** [comment]
In GitLab by @krsgpp on Jan 26, 2018, 18:40

closed via merge request !35

**Comment by valentjn** [comment]
In GitLab by @krsgpp on Jan 26, 2018, 18:40

closed via commit 09c98de99ba8b484ce1aa987f7706f5ded7c0b20

**Comment by valentjn** [comment]
In GitLab by @krsgpp on Jan 26, 2018, 18:42

mentioned in commit 09c98de99ba8b484ce1aa987f7706f5ded7c0b20
